content/docs/start/visualization.md

```diff
@@ -0,0 +1,108 @@
+# Visualization with DVC Plots
+
+In this section, we'll add visualization to [`example-dvc-experiments`][ede]
+project. If you would like to try these yourself, please refer to the project
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+One of the important plots that shows the classification performance is the
+confusion matrix. DVC uses Vega-Lite to plot the text files. In order to produce
+a confusion matrix, Vega expects a CSV file in the form:
+
+```csv
+actual,predicted
+0,0
+0,2
+...
+```
+
+We added a [straightforward loop] to the project to generate this file from the
+predictions.
+
+[straightforward loop]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L213
+
+After running the experiment with `dvc exp run`, DVC will produce
+`plots/confusion.csv` file that can immediately be shown in the browser:
+
+```dvc
+$ dvc plots show plots/confusion.csv --template confusion -x actual -y predicted
+file:///Users/.../example-dvc-experiments/plots/confusion.json.html
+```
+
+The file contains the following plot that shows the performance of the CNN
+classifier:
+
+![confusion matrix](/img/start_visualization_confusion1.png)
+
+Let's produce another plot to see misclassified examples from each class. This
+procedure is a bit more complex than creating a CSV file but you can see the
+code in [Github][misclassified-example-code]
+
+[misclassified-example-code]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L58
+
+We can see the misclassification examples in the browser:
+
+```dvc
+$ dvc plots show plots/confusion.png
+```
+
+DVC automatically shows the image in an HTML page.
+
+![confusion image](/img/start_visualization_confusion2.png)
```

Comment:
We previously said "Let's produce another plot" but this one is also called "confusion" which is 'confusing' 😋 , at least to me. Are both confusion matrices? Can we somehow clarify the situation e.g. give better names to the files?

Reply:
I guess the larger comment here is that it's not clear what the purpose of this 2nd example is. If we're trying to show that DVC supports both data-series and image files, we should state that clearly somewhere.